src/Sdk/MASA.Auth.Sdk/Request/Subjects/AddThirdPartyUserRequest.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Masa.Auth.Sdk.Request.Subjects;
+
+public class AddThirdPartyUserRequest
+{
```

Comment:
Add `public string ThridPartyIdentity { get; set; }`, used for `OpenId` or others.